### DISCUSSION TITLE
Unable to reverify academic status or organization with Github Education

### DISCUSSION THREAD

**Discussion by crinfante** [community/community#54269] [GitHub Education]
### Hi everyone,

I reverified my academic status for my personal GitHub account in 2022. I've recently received emails indicating that I need to reverify the academic status of the organization (this was last done in 2020) that I own. However, there is no link to do this on the "Get your GitHub Benefits" page, and it shows a message above the request form stating: "You were last verified as a teacher on Mar 21, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits." As far as I know, my personal academic verification is supposed to be linked to my organization verification, but this doesn't appear to be working. Does someone know the correct process to reverify an owned organization?

**Comment by whatknows** [comment]
I am running into the exact same problem. Did you ever find a solution for this?

**Comment by manathan1984** [comment]
Same problem, I wrote my full story #57451 (I guess the two comments can be merged).
